tests/prefetch_related/tests.py

```diff
@@ -256,6 +259,81 @@ def test_m2m_then_reverse_one_to_one_object_ids(self):
         self.assertWhereContains(sql, self.author1.id)
 
 
+@mock.patch('django.db.connection.features.max_query_params', 50)
+class LargePrefetchSetTests(TestCase):
+    """
+    Prefetch queries using large "WHERE id IN (…)" clauses are performed in
+    batches on databases with a limited number of variables per query, such as
+    SQLite. To improve test performance, a smaller value of max_query_params
+    is mocked.
+    """
+    test_size = 101
```

Comment:
I tried something similar but it turned out to be more annoying that I felt it was worth. I think the chances of someone changing one value and inadvertently not changing the other value is low.

As for what the test values should be... whatever works.